libs/components/lookup/src/lib/modules/lookup/lookup.component.ts

```diff
@@ -605,6 +606,8 @@ export class SkyLookupComponent
 
   public openPicker(initialSearch: string): void {
     if (this.showMoreConfig?.customPicker) {
+      this.showMorePickerId = null;
```

Comment:
Why are we using `null` here instead of `undefined`?

Reply:
Ugh, this is a bad habit I formed from an older version of Angular that required `null` to be provided to an attribute if you wanted the attribute removed from the element altogether. Updated to use `undefined` instead.